pimd/pim_cmd.c

```diff
@@ -4890,7 +5047,7 @@ DEFUN (interface_no_ip_igmp,
 				    "frr-routing:ipv4");
 }
 
-DEFUN (interface_ip_igmp_join,
+DEFUN_HIDDEN (interface_ip_igmp_join,
```

Comment:
Why are we hiding commands that people are *using* day to day?

Reply:
because now there are two ways to ad "static" joins,  `join-group` and `static-group`, those are now the two visible commands to the user just to make it clear and consist at the command line which is which. I.e:
```
ip igmp join-group...
ip igmp static-group...
```
Which I think is a better interface than 
```
ip igmp join...
ip igmp static-group...
```

The old `ip igmp join...` still works, and share the same "tab-complete" as the new `ip igmp join-group`, so in theory , the user isn't tying anything more.